src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+A [tracing tool](https://en.wikipedia.org/wiki/Tracing_(software)), on the other hand, focuses on _what_ happened (and when), rather than how many times it happened or how long it took. The resulting trace is log of events which occurred during a program's execution, often across multiple systems. Though traces most often - or, in the case of Sentry's traces, always - include timestamps, allowing durations to be calculated, measuring performance is not their only purpose. They can also show the ways in which interconnected systems interact, and the ways in which problems in one can cause problems in another.
 
+### Why Tracing?
+
+Applications typically consist of interconnected components, which are also called services. As an example, let's look at a modern web application, composed of the following components, separated by network boundaries:
+
+- Front End (Single-Page Application)
+- Backend (REST API)
+- Task Queue
+- Database Server
+- Cron Job Scheduler
+
+Each of these components may be written in a different language on a different platform. Each can be instrumented individually using a Sentry SDK to capture error data or crash reports, but that instrumentation doesn't provide the full picture, as each piece is considered separately. Tracing allows you to tie all of the data together. 
+
+In our example web application, tracing means being able to follow a request from the front end to the backend and back, pulling in data from any background tasks or notification jobs that request creates. Not only does this allow you to correlate Sentry error reports, to see how an error in one service may have propagated to another, but it also allows you to gain stronger insights into which services may be having a negative impact on your application's overall performance.
+
+Before learning how to enable tracing in your application, it helps to understand a few key terms and how they relate to one another.
+
+### Traces, Transactions, and Spans
+
+A **trace** represents the record of the entire operation you want to measure or track - like page load, an instance of a user completing some action in your application, or a cron job in your backend. When a trace includes work in multiple services, such as those listed above, it's called a **distributed trace**, because the trace is distributed across those services. To recap, a trace is a record of an operation; a distributed trace is a record of an operation that's distributed across services.
+
+Each trace consists of one or more tree-like structures called **transactions**, the nodes of which are called **spans**. In most cases, each transaction represents a single instance of a service being called, and each span within that transaction represents that service performing a single unit of work, whether calling a function within that service or making a call to a different service.
```

Comment:
@lobsterkatie at this point, would it be obvious that the trace itself is a *tree of transactions*? And ultimately a trace is a tree of spans.

Reply:
Hmmm, interesting. I totally see your point - as long as there are no orphan transactions, every trace _is_ a tree - but the only time you see all of a trace's transactions together is as a list, when you search by trace id. Perhaps that's going to change, though? I think you said you're working on a trace view, yes?

But to answer your question, no, I don't think it's necessarily obvious, because until you just pointed it out, I hadn't sat down and put 2 and 2 and 2 together to think of it that way - and I've been eating, breathing, and sleeping this stuff for two weeks now! Now that you say it, of course, it does seem obvious, but that's not how either the previous version of the docs nor this version were/are presenting it. (Our diagrams, too, only show one path in the trace tree, rather than a branching structure.)

Is this a crucial point that needs to be addressed now, before this goes out, or do you think it could be made into an Asana task for further revisions of this doc?

Reply:
> Hmmm, interesting. I totally see your point - as long as there are no orphan transactions, every trace _is_ a tree - but the only time you see all of a trace's transactions together is as a list, when you search by trace id. Perhaps that's going to change, though? I think you said you're working on a trace view, yes?

There's no concrete plans to build this trace view just yet. 

Searching by trace id is the best representation we provide on the product side.

 
> Is this a crucial point that needs to be addressed now, before this goes out, or do you think it could be made into an Asana task for further revisions of this doc?

It can be crucial. The tree of transactions is somewhat unique to Sentry compared to other tracing tools. 

The fan out becomes obvious from browser JS transactions to Python transactions by way of API calls.

Reply:
Right - JS-> server via XHR, or server -> DB via queries. If we're going to make this point, though, in the absence of any tree-y thing in the product, I think we really need to illustrate it in the docs, especially since folks easily might be missing parts of the tree when they go to try to see it live (either because they're on Team or because they don't have the right permissions), so trying to take the list and turn it into a tree in their mind could be doubly hard. 

So how about this? If it's an important point we want to get out there, I don't feel like we can do it justice right now. So I'm going to make an Asana task. But I'll also see where I can sneak in a sentence or two now as well.